### PR TITLE
Cleanup Introduction and fix video

### PR DESCRIPTION
**Pull request by benarent**
The video was broken, I've tabbed the videos so the demo is first.

image

I have also removed / tightened up the bullet points, per @stevenGravy suggestion

### CODE REVIEW

File: docs/4.3/index.md [hunk 1]
```diff
@@ -100,17 +97,6 @@ A good place to start if you want to jump right in.
...
 
-## Guides
-
-We also have several guides that go through the most typical configurations and integrations.
-
-- [Okta Integration](enterprise/sso/ssh_okta.md) - How to integrate Teleport Enterprise with Okta.
-- [ADFS Integration](enterprise/sso/ssh_adfs.md) - How to integrate Teleport Enterprise with Active Directory.
-- [One Login Integration](enterprise/sso/ssh_one_login.md) - How to integrate Teleport Enterprise with One Login.
-- [OIDC Integration](enterprise/sso/oidc.md) - How to integrate Teleport Enterprise with identity providers using OIDC/OAuth2.
-- [Kubernetes Integration](kubernetes_ssh.md) - How to configure Teleport to serve as a unified gateway for Kubernetes clusters and clusters of regular SSH nodes.
```
Comment (webvictim) [inline docs/4.3/index.md:111]: Are these back in the side nav now?

**Review by webvictim — APPROVED** [review]
LGTM with a question about link removal

**Comment by webvictim** [comment]
retest this please

File: docs/4.3/index.md (hunk 1 above)
Comment (benarent) [reply docs/4.3/index.md:111]: SSO is now in the side menu but the guides are a bit deeper. This was before the many guides we have today so it made sense to not give them as much prominence in the introduction.

**Pull request merged**